spec/request/admin/curation_ticket_spec.rb

```diff
@@ -27,7 +27,7 @@
       end
 
       specify do
-        post admin_create_curation_ticket_url(work)
+        post admin_create_curation_ticket_url(work), params: { ticket_type: 'curation' }
         expect(response).to redirect_to('Redirect Path')
```

Comment:
We should test the new logic here when a the ticket type is not 'curation'.  To make it more precise, we should also probably be testing the args passed into `admin_create_curation_ticket` or `admin_create_accessibility_ticket` using `allow(libanswer_response).to receive(...).with(...)` .